### DISCUSSION TITLE
Account flagged

### DISCUSSION THREAD

**Discussion by coutureone** [community/community#137112] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Select Topic Area

General

### Body

# Select Topic Area
Question

  Hi everyone, my friend's GitHub account [0x676e67](https://github.com/0x676e67) was flagged a week ago. My friend has been an active contributor to the open source community for nearly 5 years and has invested a lot of time and energy in various projects. However, this is the first time I have encountered such an issue.

  This suspension has severely affected our organization's private repository projects and personal open source projects. My friend submitted support ticket [#2954638](https://support.github.com/ticket/personal/0/2954638) a week ago, but unfortunately, we have not received any response yet. In addition, my friend's profile as well as the open source repository return 404 errors and are not open to the public.

  I kindly request your assistance in resolving this issue as it is critical to our ongoing work. How long does it usually take the GitHub support team to resolve such issues?
![0033018af84999811b953e816ef2f7ba](https://github.com/user-attachments/assets/b9cec487-c29e-49a4-aaec-3ab6c29c6ada)

**Comment by spronline** [comment]
It seems that the problem needs attention and response

**Comment by hosung-222** [comment]
I'm having the same problem. It hasn't been resolved for three days

**Comment by ebndev** [comment]
Hi @coutureone,

This community is managed by GitHub employees, but it is not a support forum. Your friend has done everything necessary on your end by opening a ticket on our [Support page](https://support.github.com/contact).

This is a public forum, and the community is unable to assist with nor escalate your ticket, but we can assure you it is in the right place. It will be answered in the order it was received based on our team's resources and the amount of tickets ahead of yours.

As no other Community Discussions team members will be able to provide additional help with account related questions I am going to close this discussion. We appreciate your understanding!
